### ISSUE TITLE
Uncaught TypeError: item.keywords.join is not a function

### ISSUE THREAD

**Issue by lakonis** [lexicalunit/atom-notes#25]
[Enter steps to reproduce:]

1. just trying to list/search the atom notes with the command Atom Notes Toggle

Looks like the previous error keeps coming back.

**Atom**: 1.23.3 x64
**Electron**: 1.6.15
**OS**: Ubuntu 16.04.3
**Thrown From**: [atom-notes](https://github.com/lexicalunit/atom-notes) package 1.15.5

### Stack Trace

Uncaught TypeError: item.keywords.join is not a function

```
At ~/.atom/packages/atom-notes/lib/notes-view-list.js:112

TypeError: item.keywords.join is not a function
    at NotesViewList.elementForItem (/packages/atom-notes/lib/notes-view-list.js:112:23)
    at Object.elementForItem (/packages/atom-notes/lib/notes-view-list.js:62:38)
    at /packages/atom-notes/lib/select-list-view.js:189:33)
    at Array.map (native)
    at SelectListView.renderItems (/packages/atom-notes/lib/select-list-view.js:185:23)
    at SelectListView.render (/packages/atom-notes/lib/select-list-view.js:176:12)
    at updateSync (/packages/atom-notes/node_modules/etch/lib/component-helpers.js:107:34)
    at /packages/atom-notes/node_modules/etch/lib/component-helpers.js:72:7
    at DefaultScheduler.performUpdates (/packages/project-viewer/node_modules/etch/lib/default-scheduler.js:48:34)
```

### Commands

```
  4x -3:33.8.0 editor:move-to-beginning-of-word (input.hidden-input)
  2x -3:29.8.0 editor:select-to-end-of-word (input.hidden-input)
     -3:25.1.0 editor:select-to-beginning-of-word (input.hidden-input)
     -3:24.6.0 core:select-right (input.hidden-input)
     -3:23.1.0 editor:move-to-end-of-screen-line (input.hidden-input)
  3x -3:21.6.0 editor:move-to-beginning-of-word (input.hidden-input)
     -3:19.9.0 editor:select-to-end-of-word (input.hidden-input)
     -3:18.2.0 core:backspace (input.hidden-input)
     -3:16.5.0 editor:move-to-end-of-word (input.hidden-input)
     -3:16 core:backspace (input.hidden-input)
     -3:15.5.0 editor:move-to-end-of-screen-line (input.hidden-input)
  2x -3:15.2.0 core:backspace (input.hidden-input)
  5x -2:57.4.0 editor:select-to-beginning-of-word (input.hidden-input)
     -0:16.2.0 atom-notes:toggle (input.hidden-input)
     -0:11 editor:consolidate-selections (input.hidden-input)
     -0:11 core:cancel (input.hidden-input)
```

### Non-Core Packages

```
atom-notes 1.15.5 
autocomplete-bibtex 1.1.0 
broadcast 0.4.0 
git-plus 7.10.0 
language-atom-notes 0.2.3 
language-mediawiki 2.2.0 
markdown-document 0.2.1 
markdown-folder 0.5.0 
markdown-scroll-sync 2.1.2 
markdown-toc 0.4.2 
markdown-writer 2.7.3 
nvatom 0.11.0 
pandoc-convert 1.1.1 
project-viewer 1.2.5 
spell-check-test 0.77.5 
terminal-plus 0.14.5 
tidy-markdown 3.0.1 
wordcount 2.10.4 
xml-tools 0.2.1 
Zen 0.18.0 
```

**Comment by lexicalunit** [comment]
Oops! I missed a spot in the view code that needs to guard against keywords not being an array. Sorry about that :(

**Comment by lexicalunit** [comment]
Fixed, please give v1.15.6 a try.

**Comment by lakonis** [comment]
definitely closed! :+1:
